src/app/index.html

```diff
@@ -15,7 +15,8 @@
   pageTitle="Welcome"
   layout="container"
   showBreadcrumbs="true"
-  showTableOfContents="true">
+  showTableOfContents="true"
+  #stache>
```

Comment:
The `#stache` gives the template access to the component's variables. The benefit we get is reinforcing a common pattern in stache and keeping it simple. Also, Bobby recommended we use a stache namespace in the config to prevent potential collision with other things.